### PR TITLE
Regen with v1

### PR DESCRIPTION
**Pull request by notanthony**
Modified this line and ran make. Changed the name in the go.mod to remove the /v2 suffix

https://github.com/ConductorOne/conductorone-sdk-go/blob/main/gen.yaml#L24

### CODE REVIEW

File: docs/pkg/models/shared/createpolicyrequest.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The CreatePolicyRequest message is used to create a new policy.
...
-| `PolicySteps`                                                                                                                               | map[string][shared.PolicyStepsInput](../../../pkg/models/shared/policystepsinput.md)                                                        | :heavy_minus_sign:                                                                                                                          | The map of policy type to policy steps. The key is the stringified version of the enum. See other policies for examples.                    |
+| `PolicySteps`                                                                                                                               | map[string][shared.PolicySteps](../../../pkg/models/shared/policysteps.md)                                                                  | :heavy_minus_sign:                                                                                                                          | The map of policy type to policy steps. The key is the stringified version of the enum. See other policies for examples.                    |
```
Comment (jirwin) [inline docs/pkg/models/shared/createpolicyrequest.md:12]: What is the source of these changes? Is this related to the version change?

**Review by jirwin — COMMENTED** [review]
Looks good, just have one comment.

File: docs/pkg/models/shared/createpolicyrequest.md (hunk 1 above)
Comment (notanthony) [reply docs/pkg/models/shared/createpolicyrequest.md:12]: No this is unrelated to the version change, this is just a result of regenerating the SDK. More specifically, when you have an ObjectInput the read only fields are removed. Theres also an ObjectOutput where the readonly fields are added back in.

**Review by jirwin — APPROVED** [review]
